crates/compiler/test_syntax/tests/test_fmt.rs

```diff
@@ -2967,22 +3077,58 @@ mod test_fmt {
 
             identity 43
             "
-        ));
+            ),
+            indoc!(
+                r"
+            identity = |a,
+                b,
+                # it's c!!
+                c
+                | a
+
+            identity 43
+            "
+            ),
+        );
     }
 
     #[test]
     fn closure_multiline_pattern() {
         expr_formats_same(indoc!(
             r"
-            identity = \a,
+            identity = |a,
                 b,
                 # it's c!!
                 c
-                -> a
+                | a
 
             identity 43
             "
         ));
+        expr_formats_to(
+            indoc!(
+                r"
+                identity = \a,
+                    b,
+                    # it's c!!
+                    c
+                    -> a
+
+                identity 43
+                "
+            ),
+            indoc!(
+                r"
+                identity = |a,
```

Comment:
This is okay for now, but ideally this would format to

```roc
identity = |
    a,
    b,
    # it's c!!
    c
| a
```

with the `a` pattern on its own line, and the second `|` aligned with the start of `identity`

Reply:
I agree